src/backtick/engine.clj

```diff
@@ -116,7 +130,7 @@
             (db/queue-insert! {:name (:name payload)
                                :state "running"
                                :priority (to-sql-time (time/now))
-                               :data [[]]})
+                               :data (prn-str [[]])})
```

Comment:
And here (B), I assume you checked that the data structure we're storing is the same for a cron job and a regular job with no arguments .